emarchik/02_GIT.me

```diff
@@ -0,0 +1,45 @@
+  228  git --version
```

Comment:
Could you please put this output into code section

Reply:
I don't understand what I must do.

Reply:
Done.
In addition, I changed the file extension.